rest_api/api/assets.py

```diff
@@ -53,13 +54,17 @@
 @ASSETS_BP.get('assets')
 async def get_all_assets(request):
     """Fetches complete details of all Assets in state"""
-    raise ApiNotImplemented()
+    asset_resources = await assets_query.fetch_all_asset_resources(
+        request.app.config.DB_CONN)
+    return json(asset_resources)
 
 
 @ASSETS_BP.get('assets/<name>')
 async def get_asset(request, name):
```

Comment:
`name` might come in as a url encoded string (i.e. `'Gold%20Statis'`). You'll need to decode that string before using it.

Add this import:
```
from urllib.parse import unquote
```

And then call:
```
decoded_name = unquote(name)
```

This would yield `'Gold Status'`, in the example above. You can then use `decoded_name` in place of `name`.